### PR TITLE
Add `authenticating` to the `XcodeInstallationStep`

### LINKED ISSUE #551: Loading spinner does not disappear after attempting to install when not signed in
**Describe the bug**
The loading spinner for a release does not disappear after attempting to install when not signed in.

**To Reproduce**
1. Open a fresh Xcodes install
2. Try to install a build

**Expected behavior**
For the loading spinner to disappear after dismissing the "you are not signed in" dialog.

**Screenshots**
image

**Version**
 - OS: 14.4.1
 - Xcodes: 2.1.2

### PR DESCRIPTION
**Pull request by chuganzy**
Fixes #551 

Thanks for creating this awesome app!

This PR aims to fix a minor issue where it keeps displaying the indicator on the install button on error.
To fix it, this PR introduces `authenticating` state to the `XcodeInstallationStep` and makes it to the `installing` state once "INSTALL" button is clicked.

### Before

https://github.com/user-attachments/assets/12efaa2d-51ef-42ff-8fb4-5d0035f61c14

### After

https://github.com/user-attachments/assets/ad397f66-0f9b-43ca-bc82-c36e9c6b2b4d

### CODE REVIEW

File: Xcodes/Resources/Localizable.xcstrings [hunk 1]
```diff
@@ -4446,6 +4446,23 @@
...
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Authenticating"
+          }
+        },
+        "ja" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "認証中"
```
Comment (chuganzy) [inline Xcodes/Resources/Localizable.xcstrings:4461]: Added only Japanese as I am a native speaker, but want the community help for others!

**Review by MattKiazyk — CHANGES_REQUESTED** [review]
Can we please add `Authenticating` as the value for all languages? This will be default for all languages and needs to be unfortunately added

**Comment by MattKiazyk** [comment]
Thanks @chuganzy - just one change with adding the localization values for all the languages. 🙏

**Comment by chuganzy** [comment]
@MattKiazyk Thank you for reviewing!

**Pull request merged**